src/main/java/io/github/eliux/mega/MegaSession.java

```diff
@@ -278,4 +279,51 @@ public Boolean disableHttps() {
   public Boolean isHttpsEnabled() {
     return new MegaCmdHttps().call();
   }
+
+
+  /**
+   * import exported link into your cloud
+   *
+   * @param exportedLink {@link String} the link to be imported
+   * @return {@link MegaCmdImport} to be configured and run.
+   */
+  public MegaCmdImport importLink(String exportedLink) {
+    return new MegaCmdImport(exportedLink);
+  }
+
+  /**
+   * import exported link into your cloud
+   *
+   * @param exportedLink {@link String} the link to be imported
+   * @return {@link MegaCmdImport} to be configured and run.
+   */
+  public MegaCmdImport importLinkWithPassword(String exportedLink, String password) {
```

Comment:
As you may have noticed the only required atribute is the `exportedLink` all the other attributes are optionals. So you should be able to just do
```
ImportInfo result = sessionMega.importLink("https://mega.nz/folder/etc").call();
```
and if you want to specify any other detail then just use the functions of the object `MegaCmdImport`, i.e.
```
ImportInfo result = sessionMega.importLink("https://mega.nz/folder/etc")
   .setPassword(password)
   .setRemotePath(remotePath)
   .call();
```
IMHO, this would look cleaner than creating a function for all the possible combinations with the help of `Optional` parameters.

Reply:
Thanks for suggesting this. it's actually cleaner. i will implement this instead